configs/rollup/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@toss/rollup-config",
-  "version": "0.1.4",
+  "version": "0.1.5",
```

Comment:
you don't have to increase version manually
It will increased automatically by [ci](https://github.com/toss/slash/blob/main/.circleci/config.yml)

Reply:
Thx, I restore version to original [chore: cancel version changes in rollup-config](https://github.com/toss/slash/pull/247/commits/42c43e3864a4c20a31bda338846c2ec5dcbe4f72)